README.md

```diff
@@ -185,14 +186,15 @@ The `sage-bionetworks` package will contain images for data releases that have b
 
 If a dev needs to create an image for a data release that does not yet exist in the Sage-Bionetworks package, they should follow these steps: 
 
-1. Create a new branch.
-2. Update the `package.json` to reflect the appropriate `data-file` and `data-version` files.
-3. If necessary, update `./scripts/collections.csv` to specify new collections and `./scripts/mongo-create-Indexes.js` to specify new indexes.
-4. Commit the changes.
-5. Push the changes to your remote fork to trigger a run of the `ci.yml` workflow. 
-6. The new image will be available in your user namespaced GHCR package, e.g. `https://github.com/hallieswan/Agora/pkgs/container/agora-data-nonmonorepo`.
-7. Update your local `.env` file so `DATA_IMAGE_PATH` points to the newly created image, e.g. `ghcr.io/hallieswan/agora-data-nonmonorepo:syn13363290.68`.
-8. Start the containerized database: `npm run docker:db:start`.
+1. Set up GitHub secrets/variables in your remote fork as described [here](docker/README.md#workflow-setup), if necessary.
```

Comment:
Specify that user must add creds to their GitHub fork before creating the new image. Subsequent steps are unchanged.